### ISSUE TITLE
remove old code and put into temp folder that is not a part of code coverage

### ISSUE THREAD

**Issue by jolesbi** [cosmos/cosmos-ui#462]

**Comment by faboweb** [comment]
I actually didn't find components that are not used. I used https://github.com/latentflip/webpack-unused to identify files, that are not imported.
I removed some files we no longer need in the repo in #466

**Comment by jolesbi** [comment]
```
PageDelegatePower
PageDelegateProposals
PageDelegateSlashes
PageDelegateVotes
FormGroupDelegate
```

**Comment by faboweb** [comment]
@jolesbi thx for the tip. I put a lot of the components in `archive/components`. Coverage didn't count those files in anyway. But know the file-tree is a lot cleaner.

**Comment by jolesbi** [comment]
closed by #466
